### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by zhijxu-MS**
fix several bugs

### CODE REVIEW

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -473,6 +473,7 @@ def rewrite_incomplete_type_support_rs5(g, ops):
 def rewrite_incomplete_type_support_rs6(g, ops):
     return rewrite_incomplete_type_support(g, ops, [
         "Div",
+        "Greater",
```
Comment (nbcsm) [inline tf2onnx/tfonnx.py]: how about Less?

File: tf2onnx/tfonnx.py [hunk 2]
```diff
@@ -475,6 +475,7 @@ def rewrite_incomplete_type_support_rs6(g, ops):
         "Div",
         "Greater",
         "IsNaN",
+        "Min",
```
Comment (nbcsm) [inline tf2onnx/tfonnx.py]: Max?

File: tf2onnx/rewriter/cond_rewriter.py [hunk 3]
```diff
@@ -127,7 +127,7 @@ def _get_output_shape_dtype(self, cond_context):
                         false_dtype
                     )
                 )
-            output_shapes.append(utils.merge_shapes(true_shape, false_shape))
+            output_shapes.append(utils.create_vague_shape_like(utils.merge_shapes(true_shape, false_shape)))
```
Comment (nbcsm) [inline tf2onnx/rewriter/cond_rewriter.py]: add a comment.

File: tf2onnx/tfonnx.py (hunk 1 above)
Comment (zhijxu-MS) [reply tf2onnx/tfonnx.py]: done.

**Pull request merged**